Snowflake.Data.Tests/IntegrationTests/SFDbDataReaderIT.cs

```diff
@@ -131,16 +125,15 @@ public void testGetFloat()
             {
                 conn.ConnectionString = ConnectionString;
                 conn.Open();
+                
```

Comment:
can we rename test - it binds double values and was creating testGetDouble table but after this change it will be named testGetFloat

Reply:
Done